src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -29,27 +29,59 @@ public function call($callback, array $parameters = array())
 	 * Add a new Artisan command event to the schedule.
 	 *
 	 * @param  string  $command
+	 * @param  array  $parameters
 	 * @return \Illuminate\Console\Scheduling\Event
 	 */
-	public function command($command)
+	public function command($command, array $parameters = [])
 	{
-		return $this->exec(PHP_BINARY.' artisan '.$command);
+		return $this->exec(PHP_BINARY.' artisan '.$command, $parameters);
 	}
 
 	/**
 	 * Add a new command event to the schedule.
 	 *
 	 * @param  string  $command
+	 * @param  array  $parameters
 	 * @return \Illuminate\Console\Scheduling\Event
 	 */
-	public function exec($command)
+	public function exec($command, array $parameters = [])
 	{
+		if (count($parameters))
+		{
+			$command .= ' '.$this->compileParameters($parameters);
+		}
+
 		$this->events[] = $event = new Event($command);
 
 		return $event;
 	}
 
 	/**
+	 * Compile parameters for a command.
+	 *
+	 * @param  array  $parameters
+	 * @return string
+	 */
+	protected function compileParameters(array $parameters)
+	{
+		$pieces = [];
+
+		foreach ($parameters as $key => $value)
+		{
+			if (is_numeric($key))
+			{
+				$pieces[] = $value;
+			}
+			else
+			{
+				$pieces[] = $key.'="'.$value.'"';
```

Comment:
I only know of the [`escapeshellarg`](http://php.net/manual/en/function.escapeshellarg.php) function, which is not quite what we want.

Anything else?